app/models/check_create.rb

```diff
@@ -0,0 +1,25 @@
+class CheckCreate
```

Comment:
We've got the class `CheckCreate` and private method `create_check`, which confused me slightly. I wonder if calling the model something like `CheckClaim` might be more explicit